### ISSUE TITLE
ndpiReader.exe error after compilation on Windows

### ISSUE THREAD

**Issue by mhdtbc** [ntop/nDPI#1744]
## Describe the bug
I downloaded ndpi and compiled on windows 11 through msys2 with prerequisite. ndpiReader compilation seems fine from logs, but the .exe is raising a windows error (cf. screenshot).

### Expected behavior
ndpireader.exe to execute correctly.
### Obtained behavior
error (cf. capture)
![error](https://user-images.githubusercontent.com/871101/191069430-7fbba725-a3ab-40f5-8cce-4ea5aa360502.png)

## nDPI Environment (please complete the following information):
* OS name: Windows 11
* OS version: 22H2 (build 22622.598)
* Architecture: x64
* nDPI version or commit hash: current dev branch
* no compilation flag (make)
* [config.log](https://github.com/ntop/nDPI/files/9600789/config.log)

### Reproducible using ndpiReader?
- [x] The reported bug is reproducible using ndpiReader.
- [x] The reported bug is not reproducible using ndpiReader.

### Steps to reproduce the behavior:
1. compile
2. run ndpireader.exe

**Comment by utoni** [comment]
> ### Steps to reproduce the behavior:
> 
>     1. compile

I need more details to reproduce this issue. How exactly did you compile ndpiReader?

**Comment by utoni** [comment]
@mhdtbc 
Did you install `npcap` with WinPcap compatibility mode enabled?

**Comment by mhdtbc** [comment]
Dear @utoni,

Sorry for the late answer. I followed the readme documentation to setup an environnement in my windows machine (as i need ndpireader.exe with my same windows machine wireshark). I used msys2 environnement. in my machine i have npcap for windows installed but without the winpcap compatibility mode enabled.

i run the autogen.sh script and then make in the ndpi directory.

i then tried to run ndpireader.exe from my cmd command line and i met the error above.

I'm trying to reinstall npcap with winpcap compatibility mode enabled right now.

That's it.
